libs/ledger/tests/chain/block_coordinator_tests.cpp

```diff
@@ -76,8 +76,9 @@ fetch::Digest GENESIS_DIGEST =
 fetch::Digest GENESIS_MERKLE_ROOT =
     fetch::byte_array::FromBase64("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=");
 
-constexpr std::size_t NUM_LANES  = 1;
-constexpr std::size_t NUM_SLICES = 1;
+constexpr uint32_t    LOG2_NUM_LANES = 0;
```

Comment:
why is this constrxpr 0 here

Reply:
Only because the other constant `NUM_LANES` was `constexpr`. I guess the impact is minimal in either case